app/v1/connectors/httpparsers/StandardDesHttpParser.scala

```diff
@@ -50,12 +50,11 @@ object StandardDesHttpParser extends HttpParser {
     val correlationId = retrieveCorrelationId(response)
 
     if (response.status != successCode.status) {
-      logger.info(
+      logger.error(
```

Comment:
keep it as **### info**

Reply:
this is an error though, it shouldn't be logged at info level as it will be harder to find in kibana

Reply:
We are maintaining as **info** for all our APIs. You can have a chat with the team or Mark

Reply:
Okay confirmed this with the team, thank you